### ISSUE TITLE
Help should be inline

### ISSUE THREAD

**Issue by ujmappa** [StarbeamOne/home#110]
I saw a lot of times that you link the manual, or some help page how to do this and that. But these helps should be inline with some (?) mark after a field and a description on the page. Before migrating help you should at least link the approptiate manual link on each page that users can access them directly. After migration you will not have to maintain two projects, and you will immediately notice on the page that help should be changed as well.

**Comment by Mike-E-angelo** [comment]
Nodding along here.  Is there a way to integrate Gitbook into an actual product as you suggest? 🤔

**Comment by ujmappa** [comment]
No, there isn't, or at least not that I would know of. Maybe you could integrate it as an iframe as first step? But I thought of pure HTML that contains more or less the same content, but on the page. Why would you have two places when help could be inline? It can be a continous thing, but having a separate Gitbook is for sure a bad solution from user perspective, this is not a developer manual, API description, etc.

**Comment by Mike-E-angelo** [comment]
[GitBook is what Midjourney used](https://midjourney.gitbook.io/docs/) and where I got the inspiration. :D  I see now visiting their link that they, too, evolved. 👍  I am certainly open to a better solution especially if it's cheap/free and easy to integrate.
